### DISCUSSION TITLE
B 站所有原生快捷键在焦点位于网页任何地方时都生效后续

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#2407] [功能建议 / Ideas]
https://github.com/the1812/Bilibili-Evolved/issues/1287#issuecomment-759972463 提到的 <kbd>[</kbd> 和 <kbd>]</kbd> B 站本身好像已经自带了（说实话我也已经不记得当时是怎么得出「还是不行」的结论）。

至于 <kbd>Enter</kbd>，就好像还是不行。
就是我在想如果让这个快捷键也在焦点位于网页任何地方时都生效，那是不是还得附带一个页面跳转至弹幕输入栏位置的效果（原理上类似 `自动定位到播放器` 功能？）

**Comment by the1812** [comment]
我好像没做过这 3 个的，Enter 倒可以聚焦过去，切 P 不知道该调用什么

**Reply by Geeyun-JY3** [reply]
切 P 的话我不太记得了，反正我还没碰到不能的情况，有问题到时再来这里汇报吧

**Reply by the1812** [reply]
切 P 应该要聚焦在播放器才能切吧

**Reply by Geeyun-JY3** [reply]
我试了试将焦点移到别的地方，好像还是能切的（也有可能是我的方法根本没有使焦点从播放器移开）

**Reply by the1812** [reply]
去试了下, 应该只有番剧不行

**Reply by Geeyun-JY3** [reply]
刚才没试番剧。

试了试番剧确实不行。之前可能就是因为这个。
